dotnet/v1/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mvpstudio/base:v1 AS base
+ENV ASPNETCORE_URLS=http://*:8080
+
+WORKDIR /home/mvp/app
+EXPOSE 8080
+
+# https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-debian10
+RUN apt-get install wget && \
```

Comment:
total nit-pick but it's semi-standard to indent follow-up lines when you have long multi-line things like this. Doesn't make any "real" difference - I think most people just find that easier to read.